warehouse/templates/manage/releases.html

```diff
@@ -78,6 +78,7 @@ <h2>Releases ({{ project.releases|length }})</h2>
                     View
                   </a>
                 </li>
+                {% if not disallow_deletion %}
```

Comment:
Similarly, invert to `{% if disallow_deletion %}` and provide a message visible to user.

Reply:
Using a flash instead.